line_beam_parameters.ui

```diff
@@ -2190,13 +752,2172 @@ QCheckBox::indicator:unchecked:disabled {background-color: rgb(224, 224, 224);}<
+        <number>0</number>
+       </property>
+       <property name="labels" stdset="0">
+        <stringlist>
+         <string>Bit 0</string>
+        </stringlist>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="PyDMLabel" name="jf_time_remaining">
+       <property name="minimumSize">
+        <size>
+         <width>50</width>
+         <height>26</height>
+        </size>
+       </property>
+       <property name="maximumSize">
+        <size>
+         <width>50</width>
+         <height>16777215</height>
+        </size>
+       </property>
+       <property name="toolTip">
+        <string/>
+       </property>
+       <property name="precision" stdset="0">
+        <number>2</number>
+       </property>
+       <property name="showUnits" stdset="0">
+        <bool>false</bool>
+       </property>
+       <property name="precisionFromPV" stdset="0">
+        <bool>false</bool>
+       </property>
+       <property name="alarmSensitiveContent" stdset="0">
+        <bool>false</bool>
+       </property>
+       <property name="alarmSensitiveBorder" stdset="0">
+        <bool>true</bool>
+       </property>
+       <property name="PyDMToolTip" stdset="0">
+        <string/>
+       </property>
+       <property name="channel" stdset="0">
+        <string>ca://${line_arbiter_prefix}TimeRemainJF_RBV</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QLabel" name="label_27">
+       <property name="maximumSize">
+        <size>
+         <width>125</width>
+         <height>16777215</height>
+        </size>
+       </property>
+       <property name="font">
+        <font>
+         <weight>50</weight>
+         <bold>false</bold>
+        </font>
+       </property>
+       <property name="text">
+        <string>hrs remaining</string>
+       </property>
+      </widget>
+     </item>
+    </layout>
+   </item>
+   <item row="18" column="1">
+    <widget class="QComboBox" name="rateComboBox">
+     <property name="maximumSize">
+      <size>
+       <width>200</width>
+       <height>16777215</height>
+      </size>
+     </property>
+    </widget>
+   </item>
+   <item row="31" column="0">
+    <widget class="QLabel" name="label_22">
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="toolTip">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Readback and setpoint for the maximum credible energy in the context of a transmission override. If this is 5mJ, then we will use the maximum transmission protections. If this is set to lower than 5mJ and applied, then we will adjust all transmission protections proportionally.&lt;/p&gt;&lt;p&gt;Note that if we already have a transmission override applied, adjusting this parameter further will update the protection threshold, live.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
This is kind of scary to me, being an important and (globally?) impactful setting that's hidden in a tab. Are there strict protections regarding who can write to this PV?

Reply:
This is a fair point. There are currently no real protections here. @ghalym any thoughts on this?